tools/setup-dev.sh

```diff
@@ -363,6 +363,6 @@ deactivate
 
 pushd src/glass &>/dev/null
 rm -rf node_modules
```

Comment:
You can remove this line because `npm ci` deletes this directory also.

Reply:
Thanks for the hint :)